src/js/components/Pagination/StyledPageControl.js

```diff
@@ -2,6 +2,43 @@ import styled from 'styled-components';
 
 import { Button } from '../Button';
 
+const BUTTON_STATES = ['default', 'active', 'disabled', 'hover', 'selected'];
```

Comment:
I'm not sure if `selected` is a button state we support in pagination, so I think we can remove it.

Reply:
Addressed.